### PR TITLE
Fix matching results of several chat templates like llama2, solar, yi and so on

### PR DESCRIPTION
**Pull request by AllentDan**

### CODE REVIEW

**Comment by lvhan028** [comment]
我们支持的模型都检查一下吧。
query是我们支持的所有的huggingface model repo id, 以及 model_repo_id + '-4bit'
可以再更新一下unittests

**Comment by lvhan028** [comment]
query WizardLM/WizardMath-7B-V1.1, matched name internlm-chat-7b-8k, but expected wizardlM
query WizardLM/WizardMath-70B-V1.0, matched name internlm-chat-7b-8k, but expected wizardlM

**Comment by lvhan028** [comment]
query "01-ai/Yi-34B-200K", matched name "None", but expected "yi"
query "01-ai/Yi-6B-200K", matched name "None", but expected "yi"
query "01-ai/Yi-34B-Chat", matched name "internlm-chat", but expected "yi"
query "01-ai/Yi-6B-Chat", matched name "internlm-chat", but expected "yi"
query "01-ai/Yi-34B-4bit", matched name "None", but expected "yi"
query "01-ai/Yi-6B-4bit", matched name "None", but expected "yi"
query "01-ai/Yi-34B-200K-4bit", matched name "None", but expected "yi"
query "01-ai/Yi-6B-200K-4bit", matched name "None", but expected "yi"
query "01-ai/Yi-34B-Chat-4bit", matched name "internlm-chat-7b", but expected "yi"
query "01-ai/Yi-6B-Chat-4bit", matched name "internlm-chat-7b", but expected "yi"

File: lmdeploy/model.py [hunk 1]
```diff
@@ -427,7 +427,7 @@ def messages2prompt(self, messages, sequence_start=True):
         return ret
 
 
-@MODELS.register_module(name='llama2')
+@MODELS.register_module(name=['llama2', 'llama2-chat', 'llama-2-chat'])
```
Comment (lvhan028) [inline lmdeploy/model.py:430]: llama2-chat，这个可以删掉。llama-2-chat 够用

**Review by lvhan028 — APPROVED** [review]

**Review by grimoire — APPROVED** [review]

**Pull request merged**